src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ShowAnswerCard.jsx

```diff
@@ -65,6 +65,33 @@ export const ShowAnswerCard = ({
           />
         </Form.Group>
         )}
+    </>
+  );
+
+  const explanationSection = (
+    <>
+      <div className="pb-3">
+        <span>
+          <FormattedMessage {...messages.explanationSettingText} />
+        </span>
+      </div>
+      <Form.Group className="pb-0">
+        <Form.Control
+          value={solutionExplanation}
+          onChange={handleExplanationChange}
+          floatingLabel={intl.formatMessage(messages.explanationInputLabel)}
+        />
+      </Form.Group>
+    </>
+  );
```

Comment:
Yes. I pass it in as prop instead of children because of the styling requirements. We need a divider between the showAnswerSection and the explanationSection of the showAnswerCard. Now if I add it in here, it will not have full width because it is inside the padding of the SettingsOption card.
Instead of somehow artificially increasing the width in a hacky way, this divider seemed better as a property of the settingsOption card. So I wanted to change the SettingsOption so it supports multiple body sections that are divided by this divider.